profiles/scala-2.11-spark-2-jdk-1.8.gradle

```diff
@@ -26,5 +26,5 @@ ext {
     sparkVersion="2.4.8"
     sidecarClientGroup="org.apache.cassandra.sidecar"
     sidecarClientName="vertx-client-all"
-    sidecarVersion="1.0.0-jdk8-local"
+    sidecarVersion="1.0.0.44-jdk8-local"
```

Comment:
this seems incorrect. we should use 1.0.0-jdk8-local for java8 and 1.0.0-local for jdk11

Reply:
Yifan and I changed this in hope of picking up the latest version of Sidecar client with new endpoints available. Reverted the changes, but compilator is still unhappy.